modules/config.lua

```diff
@@ -19,6 +19,8 @@ config.mobileHost = "localhost"
 config.mobilePort = 12345
 --- Define timeout for Heartbeat in msec
 config.heartbeatTimeout = 7000
+--- Define timeout to wait of unexpected events
```

Comment:
@dboltovskyi I would propose to rephrase description for this option, to it become more exact
```
--- Define timeout to wait of unexpected events => --- Define timeout to wait events which must not be raised

Reply:
@aderiabin Slightly changed in 0913260